examples/tensorflow/train_predict_2.py

```diff
@@ -0,0 +1,123 @@
+# in case this is run outside of conda environment with python2
```

Comment:
Nice, maybe add a comment to the top of this file explaining that it's a TF 2.0 example? Would also be good to update `mlflow/examples/README.md` (the section starting with "`tensorflow` is an end-to-end one run example from train to predict" isn't quite true anymore since there are two examples)

Reply:
In fact, we could even check that the user is using TF>=2.0 at the top of this file & error out / tell them to run `examples/tensorflow/train_predict.py` instead (& do the same for `examples/tensorflow/train_predict.py` if the user is using 2.0 instead of 1.x)

Reply:
IMO we can / should split out updating the examples into their own follow-up PR though, to keep this one smaller (one way would be to remove the new examples files from this PR & open a follow-up where you introduce them & address the README change suggestions^)